src/parser.cpp

```diff
@@ -0,0 +1,89 @@
+#include <iostream>
+#include <iomanip>
+#include <sstream>
+#include <fstream>
+#include <nlohmann/json.hpp>
+#include <dcmtk/dcmpstat/dcmpstat.h>
+
+using json = nlohmann::json;
+
+using namespace std;
+
+
+class Instances {
```

Comment:
the declaration of the class should go in a header file, e.g. `Parser.hpp`. And the name of the class should correspond to the name of the file, so in this case the source file name is `Parser.cpp` and the class name is `Parser`